Add 360 review guidelines

hcarreras opened the pull request:
We have growing team of engineering managment, it came the need to document how to peform 360 reviews.

mathias-abtion commented on career/360-review.md:
```diff
@@ -0,0 +1,35 @@
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
+
+There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what it is "nice" about pairing with them.
```
Would you mind curating a few resources that you've found useful so we can optimize our time?

hcarreras replied:
I'll do that. I would like to keep it outside this PR.

mathias-abtion commented on career/360-review.md:
```diff
@@ -0,0 +1,35 @@
+
+- Preparation interview: 25 minutes
+- 3 to 6 colleges interview: 25 minutes each.
+- Presenting the results: 25 minutes (this is challenging).
```
I am worrying that the person receiving the feedback will not have enough time to ask questions during the review if we do not schedule more time for this than a regular 1-on-1. What are your thoughts on this?

hcarreras replied:
I'm alright with going over for the first 1:1s. Let's gather some feedback and if we all go over time, we can extend this.

mathias-abtion replied:
Sounds good!

mathias-abtion commented on career/360-review.md:
```diff
@@ -0,0 +1,35 @@
+
+## Presenting the results
+
+The results may be overwhelming to the evaluee. People are not used to receiving such amounts of feedback in a single session. Therefore, it is crucial to present it in a way that helps the absorption. A written format can be used together with an in-person meeting explaining the results. However, be extra cautious with the written form when giving feedback. It is easy to misinterpret.
```
We should maybe note here that the person being evaluated can be offered to receive their feedback in written form before the meeting so they have a bit of time to prepare questions and digest it before the actual 360 review meeting with their direct manager.

hcarreras replied:
As long as we all understand the limitations of the written format, I'm alright with that. The EM will have enough freedom to do it as they please.

mathias-abtion commented on career/360-review.md:
```diff
@@ -0,0 +1,35 @@
+360 reviews are scheduled by upper management. The conductor and the evaluee are determined and scheduled on an ongoing basis.
+General guidelines are:
+
+- Preparation interview: 25 minutes
```
Is this the amount of time the direct manager is supposed to spend on synthesizing the feedback and preparing for the review with the person being evaluated? :-)

hcarreras replied:
You are right. I'll add `Preparing results 30 minutes`.

mathias-abtion reviewed (commented):
It is nice that you've started the process of writing this!

Apart from my comments, a few things came to my mind while reading it:
- We should perhaps note that the feedback is anonymous so the person being reviewed will not know the source of the feedback, and that the feedback itself can potentially be collected anonymously as well through an online survey.
- The 360 review is not a tool for determining raises or promotions. Its sole focus is on providing feedback.
- The direct manager needs to have a follow-up plan for how to use the feedback effectively on a day-to-day basis.

hcarreras commented:
> We should perhaps note that the feedback is anonymous so the person being reviewed will not know the source of the feedback
I've added a line to it.
>  the feedback itself can potentially be collected anonymously as well through an online survey
Good idea. We can experiment with that. Let's keep it away from the guidelines until we have experience with it.

> The 360 review is not a tool for determining raises or promotions. Its sole focus is on providing feedback.
I've added a line in goals.

> The direct manager needs to have a follow-up plan for how to use the feedback effectively on a day-to-day basis.
At the moment, the 360 reviews are not always performed by the direct manager (as I am delegating slowly). The evaluee will have the responsibility of taking the important points with their DM.

MarkusAbtion commented on career/360-review.md:
```diff
@@ -0,0 +1,38 @@
+
+## Preparation
+
+The preparation consists of getting to know which areas the evaluee wants feedback in.
+Set up a short interview 1 or 2 weeks in advance. Be clear about the intention of the meeting in the agenda. Make the evaluee prepare a list of questions. During the interview, dig a bit deeper into the list. Why do they want to know that? Who do they want to hear from? The list may include a close client and their manager. Is there anything that's not on the list but may be interesting? Use your ...
```
Some of these sentences sound like lists.
Also, I suggest changing the sentence to "The list _could_ include a close client _or a manager._ "

```suggestion
The preparation consists of getting to know which areas the evaluee wants feedback in.
Be sure to:
- Set up a short interview 1 or 2 weeks in advance. 
- Be clear about the intention of the meeting in the agenda. 
- Make the evaluee prepare a list of questions. 

During the interview, dig a bit deeper into the list: 
- Why do they want to know that? 
- Who do they want to hear from? The list could include a close client or a manager. 
Is there anything that's not on the list but may be interesting? Use your experience from other 360 reviews to suggest what the evaluee may want to know.
```

MarkusAbtion commented on career/360-review.md:
```diff
@@ -0,0 +1,38 @@
+
+## Conducting the interviews
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
```
```suggestion
Once we have the list, set up interviews with the people who work closely to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
```

MarkusAbtion replied:
```suggestion
Once we have the list, set up interviews with the people who work close to the evaluee. Invite them 1 or 2 weeks in advance and let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
```

MarkusAbtion commented on career/360-review.md:
```diff
@@ -0,0 +1,38 @@
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
+
+There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what it is "nice" about pairing with them.
```
```suggestion
There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow-up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what is "nice" about pairing with them.
```

MarkusAbtion commented on career/360-review.md:
```diff
@@ -0,0 +1,38 @@
+
+There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what it is "nice" about pairing with them.
+
+Try to balance a good mix of positive and constructive feedback. Believe me; it is hard to get constructive feedback. The evaluee will appreciate enormously the effort that you are putting into getting to the bottom of the question. How can they keep improving?
```
```suggestion
Try to balance a good mix of positive and constructive feedback. Believe me; it is hard to get constructive feedback. The evaluee will appreciate enormously the effort that you are putting into getting to the bottom of the question: How can they keep improving?
```

MarkusAbtion commented on career/360-review.md:
```diff
@@ -0,0 +1,38 @@
+
+## Presenting the results
+
+The results may be overwhelming to the evaluee. People are not used to receiving such amounts of feedback in a single session. Therefore, it is crucial to present it in a way that helps the absorption. A written format can be used together with an in-person meeting explaining the results. However, be extra cautious with the written form when giving feedback. It is easy to misinterpret. When shari ...
```
```suggestion
The results may be overwhelming to the evaluee. People are not used to receiving such an amount of feedback in a single session. Therefore, it is crucial to present it in a way that helps the absorption. A written format can be used together with an in-person meeting explaining the results. However, be extra cautious with the written form when giving feedback. It is easy to misinterpret. 
When sharing the feedback, we should not disclose who was the person giving feedback. Focus on situations, behaviors, and learning opportunities.
```

MarkusAbtion reviewed (commented):
I just have a couple of comments on some of the words.
This is a great set of instructions and guidelines for conducting a proper 360 review! 🚀
